deploy/operator.yaml

```diff
@@ -46,6 +115,7 @@ spec:
     metadata:
       annotations:
         seccomp.security.alpha.kubernetes.io/pod: runtime/default
+        container.seccomp.security.alpha.kubernetes.io/seccomp-operator: localhost/seccomp-operator-v0.1.json
```

Comment:
Seems like we should probably have deployment options for both alpha and GA seccomp (though alpha will always work for now). Also, I think we should make it easy for folks to override the operator's profile in case it breaks something for their container runtime (not sure how likely this is to happen?). Both of these could be handled in a follow-up though.

Reply:
I completely agree with the GA syntax, just thought that doing that now would mean that we either would 1) only support v1.19+ or 2) would have to maintain two deployment files. So thought that perhaps it was better to defer that decision to later on with follow-up PRs.

In terms of overriding the operator profile, that is an interesting point. Currently the operator's profile is isolated, which allows for a cluster admin to isolate it in such a way that cluster's users can't fiddle with it. This may require changes if the community has some different use cases.